test/ptp/ptp.go

```diff
@@ -262,10 +264,46 @@ var _ = Describe("[ptp]", func() {
 		var testPtpPod v1core.Pod
 		var ptpConfigSlave ptpv1.PtpConfig
 
-		BeforeEach(func() {
-			ptpConfigName := "test"
+		execute.BeforeAll(func() {
```

Comment:
Can we use the same one as in the previews context?

I will like to clean both the grandmaster and the slave and recreate them

Reply:
we can put everything from the BeforeAll in a function and call it here and in the other Context section

Reply:
Any reason we can't merge the two together then?
The test body would then be wait, delete and check (which I think makes more sense).